entrypoint.sh

```diff
@@ -41,43 +57,55 @@ chown nginx:nginx /var/cache/nginx
 mkdir -p /var/tmp/nginx
 chown nginx:nginx /var/tmp/nginx
 
+#create vhost directory
+mkdir -p /etc/nginx/vhosts/
+
+# Copy nginx main conf file.
+cp templates/nginx.conf /etc/nginx/nginx.conf
```

Comment:
Can we preserve the templating logic (`sed`, etc) for `nginx.conf` here, even though the default template doesn't use it? That way, if anyone has a customized `templates/nginx.conf` that's still a single file in a derived container image, it'll continue to work.